tools/generator/src/TargetID.swift

```diff
@@ -31,3 +31,9 @@ extension TargetID: CustomStringConvertible {
         return self.rawValue
     }
 }
+
+// MARK: Constant TargetIDs
+
+extension TargetID {
+    static let bazelDependencies = TargetID("__bazel_dependencies")
```

Comment:
Since normal `TargetID`s all have a space in them, we don't need the `__` prefix here.

```suggestion
    static let bazelDependencies = TargetID("bazel_dependencies")
```

Reply:
I added the `__` thinking that we want it to be unique and not collide with existing target IDs. Granted, it does seem unlikely that a target would come through without the slashes or colons. I will remove the prefix.

Reply:
They will always have at least one space, as as they are `"\(label) (configuration)"`, so we are free to use any non-space name for these constants.